sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileSystemAsyncClient.java

```diff
@@ -73,6 +73,8 @@
 
     public static final String ROOT_FILESYSTEM_NAME = "$root";
 
+    public static final String ROOT_DIRECTORY_NAME = "";
```

Comment:
Does this need to be part of the public API?

Reply:
It doesn't have to be. I'l change it to private.